.github/workflows/check.yaml

```
@@ -0,0 +1,40 @@
+defaults:
+  run:
+    working-directory: action-scripts
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version-file: action-scripts/.ruby-version
```

actionlint (1.7.12)

[error] input "ruby-version-file" is not defined in action "ruby/setup-ruby@v1". available inputs are "bundler", "bundler-cache", "cache-version", "ruby-version", "rubygems", "self-hosted", "token", "windows-toolchain", "working-directory"

(action)